tests/selenium/sequential-form-test.js

```diff
@@ -1,7 +1,6 @@
 module.exports = function(driver, By) {
   function test(noProvince, url) {
     var country = 'Canada';
-    driver.manage().timeouts().implicitlyWait(3000);
```

Comment:
We only need to do this once, at the start of running all tests, and not at the start of each test. Just some cleanup.